prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -555,21 +559,32 @@ class FHIRRouter(
     /**
      * Log the results of running filters (referenced by the given [filterName]) on items out of a [report] during the
      * "route" step for a [receiver], tracking the [filterType] and tying the results to a [receiver] and [bundle].
+     * @param filterName Name of evaluated filter
+     * @param bundle FHIR Bundle that was evaluated
+     * @param report Report object passed for logging purposes
+     * @param receiver Receiver of the report
+     * @param filterType Type of filter used
+     * @param focusResource Starting point for the evaluation, can add additional data to the log
```

Comment:
If you take my suggestion my other comment, then I'd suggest a slight rewording here.
```suggestion
     * @param focusResource Starting point for the evaluation, logged if different than [bundle]
```